app/controllers/events_controller.rb

```diff
@@ -53,7 +53,7 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_places
-      @place = Place.all
+      @places = Place.all
```

Comment:
If you do only this change and application is not failing then you do not use this variable or you didn't test it enough. You should also adjust your templates (I mean the html files)